docs/assets/js/swagger.yml

```diff
@@ -2024,6 +2024,10 @@ paths:
         name: branch
         required: true
         type: string
+      - in: query
+        description: will change export status from export-failed to repaired and start a new export
+        name: repair
+        type: boolean
```

Comment:
This doesn't feel like a boolean value to me, but rather like a different command.  So it should either be an enum ("`action`"?) or an entirely separate endpoint.